### PR TITLE
SC-6684 - text change

### PR DESCRIPTION
**Pull request by ABialoskorski**
# Description
text changes

## Links to Tickets or other pull requests

- https://ticketsystem.schul-cloud.org/browse/SC-6684

## Screenshots of UI changes
image
image
image

### CODE REVIEW

**Review by mwolowyk-cap — APPROVED** [review]

File: locale/en.json [hunk 1]
```diff
@@ -283,9 +283,9 @@
...
 	"pages.administration.students.infobox.paragraph-4": "IMPORTANT: For students under 16 years of age, the first thing asked for during the registration process is the consent of a parent or guardian. In this case, we recommend the distribution of registration links as QR-sheets or retrieval of the links individually and sending them to the parents. After the parents have given their consent, the students receive a start password and can complete the last part of the registration on their own at any time. For use of the school cloud at primary schools, paper forms are a popular alternative.",
-	"pages.administration.students.legend.icon.danger": "Keine Registrierungen vorhanden",
-	"pages.administration.students.legend.icon.success": "Alle Registrierungen vorhanden",
-	"pages.administration.students.legend.icon.warning": "Parents have approved",
+	"pages.administration.students.legend.icon.danger": "User created",
+	"pages.administration.students.legend.icon.success": "Registration complete",
+	"pages.administration.students.legend.icon.warning": "Student have not approved",
```
Comment (dominikbruechner) [inline locale/en.json:288]:
```suggestion
	"pages.administration.students.legend.icon.warning": "Student has not consented",
```

File: locale/en.json [hunk 2]
```diff
@@ -375,9 +375,9 @@
 	"storybook.lang": "I'm in English now!",
 	"utils.adminFilter.class.title": "Class(es)",
 	"utils.adminFilter.consent": "Consent form:",
-	"utils.adminFilter.consent.label.missing": "No registrations available",
-	"utils.adminFilter.consent.label.ok": "All registrations available",
-	"utils.adminFilter.consent.label.parentsAgreed": "Parents have agreed",
+	"utils.adminFilter.consent.label.missing": "User created",
+	"utils.adminFilter.consent.label.ok": "Registration complete",
+	"utils.adminFilter.consent.label.parentsAgreed": "Student have not approved",
```
Comment (dominikbruechner) [inline locale/en.json:380]:
```suggestion
	"utils.adminFilter.consent.label.parentsAgreed": "Student has not approved",
```
or
```suggestion
	"utils.adminFilter.consent.label.parentsAgreed": "Student has not consented",
```

File: locale/en.json (hunk 1 above)
Comment (ABialoskorski) [reply locale/en.json:288]: done, thanks ;)

File: locale/en.json (hunk 2 above)
Comment (ABialoskorski) [reply locale/en.json:380]: done :) thanks

File: locale/de.json [hunk 3]
```diff
@@ -375,9 +375,9 @@
 	"storybook.lang": "Ich bin jetzt auf Deutsch!",
 	"utils.adminFilter.class.title": "Klasse(n)",
 	"utils.adminFilter.consent": "Einverständniserklärung:",
-	"utils.adminFilter.consent.label.missing": "Keine Registrierungen vorhanden",
-	"utils.adminFilter.consent.label.ok": "Alle Registrierungen vorhanden",
-	"utils.adminFilter.consent.label.parentsAgreed": "Eltern haben zugestimmt",
+	"utils.adminFilter.consent.label.missing": "Nutzer:in angelegt",
+	"utils.adminFilter.consent.label.ok": "Registrierung abgeschlossen",
+	"utils.adminFilter.consent.label.parentsAgreed": "Schülereinverständnis fehlt",
```
Comment (DanielRieckhof) [inline locale/de.json:380]: `parentsAgreed` is the label name for missing agreement? ;-)

File: locale/en.json [hunk 4]
```diff
@@ -375,9 +375,9 @@
 	"storybook.lang": "I'm in English now!",
 	"utils.adminFilter.class.title": "Class(es)",
 	"utils.adminFilter.consent": "Consent form:",
-	"utils.adminFilter.consent.label.missing": "No registrations available",
-	"utils.adminFilter.consent.label.ok": "All registrations available",
-	"utils.adminFilter.consent.label.parentsAgreed": "Parents have agreed",
+	"utils.adminFilter.consent.label.missing": "User created",
+	"utils.adminFilter.consent.label.ok": "Registration complete",
+	"utils.adminFilter.consent.label.parentsAgreed": "Student has not approved",
```
Comment (DanielRieckhof) [inline locale/en.json:380]: I think, it should be more like "No student consent", or "Student consent missing"

File: locale/de.json (hunk 3 above)
Comment (DanielRieckhof) [reply locale/de.json:380]: Proposal: "studentConsentMissing" or "parentsAgreementMissing"

Comment (ABialoskorski) [reply locale/de.json:380]: good ;) done

File: locale/en.json (hunk 4 above)
Comment (ABialoskorski) [reply locale/en.json:380]: this text comes from Domi, so UX perspective, I'll stick with his idea

**Review by monaweitzenberg — APPROVED** [review]
thx artur. everything is fine!!! :)

**Review by mwolowyk-cap — APPROVED** [review]

**Pull request merged**